lib/packwerk/commands/update_deprecations_command.rb

```diff
@@ -0,0 +1,67 @@
+# typed: true
```

Comment:
If we are introducing new files in the codebase I think it makes sense to make them strict by default unless it requires a lot of refactoring or is a test case.

Reply:
https://github.com/Shopify/packwerk/pull/78/commits/1b2b7f8ee983bdf18a6f3fad510b4cd7276b5582#diff-e2c1abfdf20fdf0a35e15eefaf378b134a2274fd170ae63865b041bfec4ed1b0R56

This change was to prevent sorbet complaining about memoization, but now when running tests I get

`lib/packwerk/commands/update_deprecations_command.rb:55: warning: instance variable @run_context not initialized`

any ideas?